models/Importer.js

```diff
@@ -0,0 +1,30 @@
+import csv from 'csvtojson';
+
+export default class Importer {
+	constructor (eventEmitter) {
+		this.eventEmitter = eventEmitter;
+		eventEmitter.on('dirWathcer:changed', (filePath) => this.import(filePath));
+	}
+
+	import (path) {
+		return new Promise((resolve) => {
+			csv()
+			.fromFile(path)
```

Comment:
If you're using the chaining like this, please, make one indentation after the first method call, e.g.

meth()
--.child1()
--.child2()